config/src/content/jcr_root/apps/core/wcm/config/com.adobe.cq.wcm.core.components.internal.services.cloudviewer.CloudViewerConfigServiceImpl.config

```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 Adobe
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+clientId="28b07c6be4d544f188dc2f36e33b196b"
```

Comment:
I'm not comfortable with shipping this as a default configuration. If we switch to the context-aware config we can provide this id for the examples.